### DISCUSSION TITLE
How to set number of MySQL connections? Or how to see number of MySQL connections in Budibase?

### DISCUSSION THREAD

**Discussion by dhfyes** [Budibase/budibase#6089] [Help] [answered]
How to set number of MySQL connections? Or how to see number of MySQL connections in Budibase? how can help me and answer the question，thanks

**Comment by melohagan** [comment]
Hi @dhfyes 

I'm not quite sure what you mean. 

You can see the number of client connections in MySQL workbench. The user and host should tell you if it's coming from Budibase.

**Reply by dhfyes** [reply]
When querying through mysql, the client will call multiple queries concurrently, and each query takes a long time. However, if only one query is called, the speed is very fast. I suspect that the query is slow due to the limit of the number of database connections, but I cannot view the configuration of the number of database connections in budibase

**Reply by dhfyes** [reply]
Hi @melohagan 
Query 20 requests at the same time, dose each request can create a new mysql database connetion？I want to see the configuration of database connection pool in budibase, but i can't find it

**Reply by melohagan** [reply]
I've had a look at the code and it appears that each request does create a new connection. We use this client: https://github.com/sidorares/node-mysql2

Every query calls `createConnection` but there's no mention of `createPool` (https://github.com/sidorares/node-mysql2#using-connection-pools) in the codebase.

**Reply by dhfyes** [reply]
@melohagan Thank you very much

**Comment by dhfyes** [comment, marked as answer]
Now, use createConnection method to call mysql in budibase, and no environment variable settings
